paper/paper.md

```diff
@@ -0,0 +1,103 @@
+---
+title: 'OceanBioME.jl: A flexible environment for modelling the coupled interactions between ocean biogeochemistry and physics'
+tags:
+  - julia
+  - biogeochemistry
+  - climate
+  - ocean
+  - carbon
+authors:
+  - name: Jago Strong-Wright
+    orcid: 0000-0002-7174-5283
+    corresponding: true
+    affiliation: "1, 2"
+  - name: John R Taylor
+    affiliation: "1, 2"
+  - name: Si Chen
+    affiliation: "1, 2"
+  - name: Gregory LeClaire Wagner
+    orcid: 0000-0001-5317-2445
+    affiliation: 3
+  - name: Collaborators
+affiliations:
+ - name: Department of Applied Mathematics and Theoretical Physics, University of Cambridge, Cambridge, United Kingdom
+   index: 1
+ - name: Centre for Climate Repair, Cambridge, United Kingdom
+   index: 2
+ - name: Massachusetts Institute of Technology
+   index: 3
+date: 15 March 2023
+bibliography: paper.bib
+---
+
+# Summary
+
+``OceanBioME.jl`` is a flexible modelling environment written in Julia [@julia] for modelling the coupled interactions between ocean biogeochemistry, carbonate chemistry, and physics.
+OceanBioME can be used as a stand-alone box model, or integrated into ``Oceananigans.jl`` [@Oceananigans] simulations of ocean-flavoured fluid dynamics in one-, two-, or three-dimensions.
+As a result, OceanBioME and Oceananigans can be used to simulate the biogeochemical response across an enormous range of scales: from surface boundary layer turbulence at the meter scale to eddying global ocean simulations at the planetary scale, and on computational systems ranging from laptops to supercomputers.
+An example small-scale problem is shown in Figure 1.
```

Comment:
```suggestion
An example small-scale problem is shown in \autoref{fig1}.
```

Reply:
This sentence reads a bit as if the problem in figure 1 is small and not that as it involves small scale flow features. Perhaps rephrase to "An example of a problem involving kilimeter-lenght flow features is shown..."?